### PR TITLE
Improve whitespace for Parser tests

### PR DESCRIPTION
**Pull request by vexx32**
# PR Summary

Initial part of #9792. Splitting this out to simplify review process. 🙂 

## PR Context

Parser tests are really difficult to read through and work with. We don't need to edit them a lot, but when we do it tends to be a test of patience, and we can make it less of a hassle.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

**Comment by PoshChan** [comment]
@vexx32, your last commit had 4 failures in `PowerShell-CI-windows`
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1:1143 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1:1143 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1:1143 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Parser\Parser.Tests.ps1:1143 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```

**Comment by PoshChan** [comment]
@vexx32, your last commit had 4 failures in `PowerShell-CI-macos`
Error occurred in test script '/Users/vsts/agent/2.150.3/work/1/s/test/powershell/Language/Parser/Parser.Tests.ps1'
```powershell
At /Users/vsts/agent/2.150.3/work/1/s/test/powershell/Language/Parser/Parser.Tests.ps1:1143 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/vsts/agent/2.150.3/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.150.3/work/1/s/test/powershell/Language/Parser/Parser.Tests.ps1'
```powershell
At /Users/vsts/agent/2.150.3/work/1/s/test/powershell/Language/Parser/Parser.Tests.ps1:1143 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/vsts/agent/2.150.3/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.150.3/work/1/s/test/powershell/Language/Parser/Parser.Tests.ps1'
```powershell
At /Users/vsts/agent/2.150.3/work/1/s/test/powershell/Language/Parser/Parser.Tests.ps1:1143 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/vsts/agent/2.150.3/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.150.3/work/1/s/test/powershell/Language/Parser/Parser.Tests.ps1'
```powershell
At /Users/vsts/agent/2.150.3/work/1/s/test/powershell/Language/Parser/Parser.Tests.ps1:1143 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/vsts/agent/2.150.3/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```

**Comment by vexx32** [comment]
Whoopsie, let me double check what I did there...

File: test/powershell/Language/Parser/Parser.Tests.ps1 [hunk 1]
```diff
@@ -2,75 +2,73 @@
 # Licensed under the MIT License.
 Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)" -Tags "CI" {
```
Comment (iSazonov) [inline test/powershell/Language/Parser/Parser.Tests.ps1]: Please add new line.

File: test/powershell/Language/Parser/Parser.Tests.ps1 [hunk 2]
```diff
@@ -985,102 +972,111 @@ foo``u{2195}abc
...
-			2 ) /
-			2 -
-			3"; Expected = "2" }
+            @{
+                Script   = '$a=6; $a -= 2;$a'
+                Expected = 4
+            }
+            @{
+                Script   = "20 %
+            6"
```
Comment (iSazonov) [inline test/powershell/Language/Parser/Parser.Tests.ps1]: Indent.

File: test/powershell/Language/Parser/Parser.Tests.ps1 [hunk 3]
```diff
@@ -1143,4 +1139,4 @@ foo``u{2195}abc
             $tokens[1] | Should -BeExactly $lastToken
         }
     }
-}
+}
```
Comment (iSazonov) [inline test/powershell/Language/Parser/Parser.Tests.ps1]: Please add newline at EOL.

**Review by iSazonov — APPROVED** [review]
@vexx32 Thanks for the cleanup!

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**